Templates/AppSource App/.github/workflows/CICD.yaml

```diff
@@ -95,6 +98,7 @@ jobs:
         with:
           shell: powershell
           maxBuildDepth: ${{ env.workflowDepth }}
+          publishSkippedProjectArtifacts: 'true'
```

Comment:
Why is this set on the workflow when is dynamically determined, based on the GitHub event name?

Also, why not always publish the skipped project artifacts? That way, it will be way easier to "publish a PR to an environment", for instance.

Reply:
Removed this one - artifacts are always published for this exact reason (publish a PR)